gulpfile.js

```diff
@@ -27,6 +27,8 @@ var DOCS = path.join(SRC, 'docs');
 var SASS = path.join(SRC, 'sass');
 var COMPONENTS = path.join(SRC, 'components');
 
+var svgPolyfill = fs.readFileSync(path.join(SRC, 'svg-polyfill.js'), "utf8");
```

Comment:
I think we should move it inside the task not to block gulp start (so this would probably require joining those 2 tasks for icons/subject generation)